src/components/missions.css

```diff
@@ -0,0 +1,41 @@
+table,
+tr,
+td {
+  border: 1px solid gainsboro;
+}
+
+table {
+  border-collapse: collapse;
+  margin: 5%;
+}
+
+.bold {
+  font-weight: bold;
+  padding: 1%;
+  width: 10%;
+}
+
+td {
+  padding: 1%;
+}
+
+.desc {
+  width: 60%;
+}
+
+.buttons {
+  width: 12%;
+}
+
+.join {
+    background: none;
+    padding: 5%;
+    width: 90%;
+}
+
+.leave {
+    background: none;
+    width: 90%;
+    padding: 5%;
+    border: 2px solid crimson;
+}
```

Comment:
- You can group padding and background for buttons into one selector to comply with DRY concept.